.ci/pipelines/e2e-tests-testmutationsecondmastersetdown.Jenkinsfile

```diff
@@ -0,0 +1,46 @@
+// This library overrides the default checkout behavior to enable sleep+retries if there are errors
+// Added to help overcome some recurring github connection issues
+@Library('apm@current') _
+
+pipeline {
+
+    agent {
+        label 'linux'
+    }
+
+    options {
+        timeout(time: 24, unit: 'HOURS')
+    }
+
+    environment {
+        VAULT_ADDR = credentials('vault-addr')
+        VAULT_ROLE_ID = credentials('vault-role-id')
+        VAULT_SECRET_ID = credentials('vault-secret-id')
+        GCLOUD_PROJECT = credentials('k8s-operators-gcloud-project')
+    }
+
+    stages {
+        stage('Checkout from GitHub') {
+            steps {
+                checkout scm
+            }
+        }
+        stage("E2E tests") {
+            steps {
+                sh '.ci/setenvconfig e2e/master'
```

Comment:
If this job catches up with the master build at some point, k8s cluster names could conflict.

Reply:
Yes, I know.

Reply:
I may have a quickfix.